### ISSUE TITLE
Backitems

### ISSUE THREAD

**Issue by riNdes** [Demigod916/ox_inventory_addons#8]
Hello, when a Police Officer enters a vehicle, and then exit the vehicle the weapon is not showing on the Police Officer before they pull out the weapon, do you know what that could be?

**Comment by Demigod916** [comment]
> Hello, when a Police Officer enters a vehicle, and then exit the vehicle the weapon is not showing on the Police Officer before they pull out the weapon, do you know what that could be?

I assume you are on v1.1.0?

**Comment by Demigod916** [comment]
Can you update to latest release first? ![image](https://github.com/Demigod916/ox_inventory_addons/assets/60995067/5cd9591a-31f0-4c82-8283-64ccd9677c98)

**Comment by riNdes** [comment]
> Can you update to latest release first? ![image](https://private-user-images.githubusercontent.com/60995067/287461831-5cd9591a-31f0-4c82-8283-64ccd9677c98.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE1MzUzODcsIm5iZiI6MTcwMTUzNTA4NywicGF0aCI6Ii82MDk5NTA2Ny8yODc0NjE4MzEtNWNkOTU5MWEtMzFmMC00YzgyLTgyODMtNjRjY2Q5Njc3Yzk4LmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIwMlQxNjM4MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZjRjZjIyMzNjYmExZmEyNjEyYTk4MGY5OThmNWRlMGQ1Yjg4OWMwMDZhMWZiMDkxNmUzODM1NGQ1YjU4MzM0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.DlgtoPVWZA_KHjtC-gY5yYBaSPP2HzReNSOj971wdyE)

Yes i can try and see if that fixes the issue.

**Comment by riNdes** [comment]
@Demigod916 When i download v1.1.0 i get version '1.0.9' in fxmanifest.lua?

**Comment by Demigod916** [comment]
That is likely just a mistake on my end when creating the release. Everything else should be correct code wise

**Comment by Demigod916** [comment]
What framework are you using? 

What is your inventory version?

**Comment by riNdes** [comment]
> That is likely just a mistake on my end when creating the release. Everything else should be correct code wise

Well i don't think anything is correct code wise:

I bumped from version 1.0.8 to 1.0.9 

And the only changes is this;
https://i.imgur.com/iv0TVfB.png

**Comment by riNdes** [comment]
> What framework are you using?
> 
> What is your inventory version?
> 
> Im stepping out for a bit, but when i get back home i will investigate this

Framework: QB-core
Inventory version: version '2.35.1'

**Comment by Demigod916** [comment]
> > That is likely just a mistake on my end when creating the release. Everything else should be correct code wise
> 
> 
> 
> Well i don't think anything is correct code wise:
> 
> 
> 
> I bumped from version 1.0.8 to 1.0.9 
> 
> 
> 
> And the only changes is this;
> 
> https://i.imgur.com/iv0TVfB.png

Yeah 1.0.9 and 1.1.0 were mainly fixes/features for carryItems. I just wanted to ensure that it was still not working with the latest
